packages/x-data-grid/src/components/virtualization/GridVirtualScrollbar.tsx

```diff
@@ -71,7 +72,15 @@ const ScrollbarHorizontal = styled(Scrollbar)({
   '& > div': {
     height: 'var(--size)',
   },
-  bottom: '0px',
+  bottom: 0,
+});
+
+export const ScrollbarCorner = styled(Scrollbar)({
```

Comment:
This corner piece covers up the dead zone between the two scrollbars.

| Without | With |
| --- | --- |
| <img width="628" alt="Screenshot 2025-05-21 at 17 01 47" src="https://github.com/user-attachments/assets/57057576-d60c-4765-952d-6688d908fb00" /> | <img width="627" alt="Screenshot 2025-05-21 at 17 01 16" src="https://github.com/user-attachments/assets/12faac50-be8c-468d-8e2c-c0e52ef6bab9" /> |